_ci/publish-amis-in-new-account.md

```diff
@@ -4,7 +4,7 @@ This readme discusses how to migrate the `publish-amis.sh` script to a new AWS a
 
 To make using this Module as easy as possible, we want to automatically build and publish AMIs based on the 
 [/examples/consul-ami/consul.json](https://github.com/hashicorp/terraform-aws-consul/tree/master/examples/consul-ami/consul.json) Packer template upon every release of this repo. 
-This way, users can simply git clone this repo and `terraform apply` the [consul-cluster](https://github.com/hashicorp/terraform-aws-consul/tree/master/MAIN.md)
+This way, users can simply git clone this repo and `terraform apply` the [consul-cluster](https://github.com/hashicorp/terraform-aws-consul/tree/master/modules/consul-cluster)
```

Comment:
So, this is one of the confusing requirements of HashiCorp's Terraform Registry: every repo _must_ have a `main.tf` in the root of the repo. Since the Consul code requires multiple modules, we couldn't put them in the root, and instead, the root folder is just example code for how to use Consul. 

The `MAIN.md` link is probably some copy/paste mess up, but it should actually point to the root example in this repo. We created a folder in `examples` to describe this: https://github.com/hashicorp/terraform-aws-consul/tree/master/examples/root-example. Perhaps adding this note there for explanation would help too.